### PR TITLE
Update Powerschool URL

### PR DESCRIPTION
**Pull request by mud-ali**
Fix for the new powerschool url migration, as per #3.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name        BCA GPA
 // @description GPA Calculator for BCA
-// @match       https://ps001.bergen.org/*
+// @match       https://bcts.powerschool.com/*
 // ==/UserScript==
 
-if (document.URL === "https://ps001.bergen.org/guardian/home.html") {
+if (document.URL === "https://bcts.powerschool.com/guardian/home.html") {
```
Comment (kna27) [inline src/main.js]: Doesn't work if user signs in through the Student Sign In page. This adds `?_userTypeHint=student` to the end of the URL, so could you update this to account for any URL parameters?

**Review by kna27 — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (mud-ali) [reply src/main.js]: updated and tested [with edge chromium]

**Review by kna27 — CHANGES_REQUESTED** [review]
Still doesn't work, change match URL in the manifest.json files to `"https://bcts.powerschool.com/guardian/*"`.

**Comment by mud-ali** [comment]
Apparently sometimes powerschool redirects to 
`https://bcts.powerschool.com/guardian/home.html?_userTypeHint=student`
and sometimes to 
`https://bcts.powerschool.com/guardian/home.html/?_userTypeHint=student`

**Review by kna27 — APPROVED** [review]

**Pull request merged**